go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	connectrpc.com/grpchealth v1.3.0
 	github.com/AlecAivazis/survey/v2 v2.3.7
 	github.com/Masterminds/semver/v3 v3.2.1
+	github.com/NYTimes/gziphandler v1.1.1
```

Comment:
This library does not appear to be actively maintained, a fork that does appear to be actively maintained is https://github.com/klauspost/compress/tree/master/gzhttp